karma.shared.js

```diff
@@ -2,16 +2,19 @@
 
 /* This is the shared karma config */
 
+var merge = require('lodash.merge');
 var RewirePlugin = require('rewire-webpack');
-var webpackConfig = require('./webpack.config.js');
+var webpackConfig = require('./webpack.config');
 
+
+var newWebpackConfig = merge({}, webpackConfig);
```

Comment:
Also as discussed, carry on! Maybe we could just remove assign everywhere for now? A quick grep suggests to me that we aren't even using it, at least not in prod code.